L1Trigger/RPCTechnicalTrigger/src/RPCTechnicalTrigger.cc

```diff
@@ -454,25 +394,24 @@ int RPCTechnicalTrigger::convertToMap( const std::vector<TTUResults*> & ttuResul
     
     int key(0);
     key = 1000 * ( (*itr)->m_ttuidx + 1 ) + 1*(*itr)->m_wedge;
-    m_ttuResultsByQuadrant[ key ] = (*itr);
+    returnValue[ key ] = itr->get();
     ++itr;
-    
   }
   
-  return m_ttuResultsByQuadrant.size();
+  return returnValue;
     
 }
 
 //...RBC pointing logic to tracker bit 25: hardwired
-bool RPCTechnicalTrigger::searchCoincidence( int wheel1, int wheel2 )
+bool RPCTechnicalTrigger::searchCoincidence( int wheel1, int wheel2,  std::map<int, TTUResults*> const& ttuResultsByQuadrant) const
 {
   
-  std::map<int, TTUResults*>::iterator itr;
+  std::map<int, TTUResults*>::const_iterator itr;
   bool topRight(false);
   bool botLeft(false);
   
-  int indxW1 = m_WheelTtu[wheel1];
-  int indxW2 = m_WheelTtu[wheel2];
+  int indxW1 = wheelTtu[wheel1+kWheelOffset];
```

Comment:
Previously a map was being used. But the keys were just {-2,-1,0,1,2}. So a `std::vector` with an offset can do the job better.